osbs/constants.py

```diff
@@ -102,6 +102,10 @@
 BUILD_TYPE_WORKER = "worker"
 
 ISOLATED_RELEASE_FORMAT = re.compile(r'^\d+\.\d+(\..+)?$')
+RELEASE_LABEL_FORMAT = re.compile(r"""^\d+             # First character must be a digit
+                                      ([._]?           # allow separators between groups
+                                      [a-zA-Z0-9]+)*$  # last characters must be alphanumeric""",
```

Comment:
I really enjoy this way of wring regexes, much easier to read!

A minor improvement would be to move the closing `""",` to a new line.

Reply:
sure